### ISSUE TITLE
Accessibility: Problems with ARIA child roles "Certain ARIA roles must contain particular children"

### ISSUE THREAD

**Issue by MarkoTP** [microsoft-search/pnp-modern-search#4025]
**Version used**
4.13.1

**Describe the bug**
Accessibility:
A few PnP Modern Search issues were identified during the accessibility audit.
Axe DevTools informs of problems with ARIA child roles in the PnP Modern Search Filter web part: "Certain ARIA roles must contain particular children".  Ensure elements with an ARIA role that require child roles contain them.
Axe DevTools: [axe Browser Extensions for Accessibility Testing | Deque](https://www.deque.com/axe/browser-extensions/)
WCAG 2.1, 1.3.1

**To Reproduce**
Detailed steps to reproduce the behavior:
Configure search page you like with search box, results and filters web parts. There should be at least one filter defined.
Execute a search with keyword that returns filterable results.
Execute the axe DevTools accessibility tool on the page.
The axe DevTools tool should find multiple "Certain ARIA roles must contain particular children" -errors in the filter web part and provide suggestions for fixing them.

**Expected behavior**
The axe DevTools should not find any technical errors in the web part.

**Desktop (please complete the following information):**
Browser: Chrome, Version 129.0.6668.90 (Official Build) (64-bit)
